### PR TITLE
Add ability to delete an issue comment

### PR DESCRIPTION
**Pull request by chteuchteu**
For now, we can only add and update comments.
This PR adds the ability to delete a comment.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (radutopala) [inline .idea/vcs.xml]: @chteuchteu pls remove this file from the commit

**Comment by chteuchteu** [comment]
Just rebased my PR to remove the nasty .idea/vcs.xml file that slipped into it

File: lib/Gitlab/Api/Issues.php [hunk 2]
```diff
@@ -105,4 +105,15 @@ public function updateComment($project_id, $issue_id, $note_id, $body)
...
         ));
     }
+
+    /**
+     * @param int $project_id
+     * @param int $issue_id
+     * @param int $note_id
+     * @return mixed
+     */
+    public function deleteComment($project_id, $issue_id, $note_id)
```
Comment (radutopala) [inline lib/Gitlab/Api/Issues.php]: @chteuchteu Pls rename to `removeComment`, to have consistency compared with other methods within the project. See #149 discussion.

Comment (chteuchteu) [inline lib/Gitlab/Api/Issues.php]: Done!

Comment (brammeleman) [inline lib/Gitlab/Api/Issues.php]: Thanks!

**Pull request merged**